packages/components/Avatar/src/Avatar.styling.ts

```diff
@@ -151,14 +153,32 @@ export const stylingSettings: UseStylingOptions<AvatarProps, AvatarSlotProps, Av
             minHeight: ringConfig.size,
             ...borderStyles.from(tokens, theme),
             borderWidth: ringConfig.ringThickness,
-            backgroundColor: ringBackgroundColor || 'transparent',
             borderColor: ringColor,
+            ...getRingSpacing(tokens),
+            backgroundColor: ringBackgroundColor || 'transparent',
             aspectRatio: 1,
           },
         };
       },
       ['size', 'ringColor', 'ringBackgroundColor', 'ringThickness', ...borderStyles.keys],
     ),
+    outerRing: buildProps(
```

Comment:
is outer ring also needed on other platforms ? just a question 🤔  

if not we can do something like this :  ...(Platform.OS === 'android' && outerRing : () )

Reply:
This is not needed in win32, updated the check.